src/scene_server/topo_server/service/inst_business.go

```diff
@@ -102,28 +95,32 @@ func (s *Service) CreateBusiness(ctx *rest.Contexts) {
 
 // UpdateBusiness update the business
 func (s *Service) UpdateBusiness(ctx *rest.Contexts) {
-	data := mapstr.MapStr{}
+	data := make(mapstr.MapStr, 0)
```

Comment:
map的make不需要加长度，下面的也改下
```suggestion
	data := make(mapstr.MapStr)
```